lib/build/bundled-source.js

```diff
@@ -214,7 +214,7 @@ exports.BundledSource = class {
         let contents;
         // forceCjsWrap bypasses a r.js parse bug.
         // See lib/amodro-trace/read/cjs.js for more info.
-        let forceCjsWrap = !!modulePath.match(/\/(cjs|commonjs)\//i);
+        let forceCjsWrap = !!modulePath.match(/(\/|\\)(cjs|commonjs)(\/|\\)/i);
```

Comment:
I'm not sure what's the best practice here. In theory this would accept a mix of both forward and backward slashes, which might not even be a problem.

Alternatively we could use the OS native slash style.

```javascript
const sep = require("path").sep === "\\"
  ? "\\\\"
  : "/"
const target = "/cjs/";

const res = target.match(new RegExp(`(${sep})(cjs|commonjs)(${sep})`, "i"))

console.log(res);
```

Reply:
I thought about that when I was doing the fix. The lazy side of me prevailed:-)

Reply:
Hehe, yeah I think we're fine with your approach